trade-finance-manager-ui/templates/feedback/feedback-thankyou.njk

```diff
@@ -0,0 +1,11 @@
+{% extends "index.njk" %}
+
+{% block pageTitle %}Feedback{% endblock %}
+
+{% block content %}
+
+  <h1 class="govuk-heading-l govuk-!-padding-top-5" data-cy="thank-you-heading">Feedback</h1>
+
+  <p class="govuk-body" data-cy="thankyou-copy">Thank you for your feedback. We will use your feedback for future improvement. Do let us know if there is anything else we need to know to improve this area of concern.</p>
```

Comment:
it looks like we have an inconsistent naming here

we are using `thank-you-heading` but also `thankyou-copy`. can we standardise it?

Reply:
done